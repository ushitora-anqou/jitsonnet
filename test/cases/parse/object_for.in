{
  local v1 = "s",
  [x]: "y",
  local v2 = "s",
  for x in a
  for y in a
  if true
}
