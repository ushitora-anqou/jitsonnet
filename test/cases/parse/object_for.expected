{ Syntax.v =
  (Syntax.Object
     (Syntax.ObjectFor
        ([(Syntax.Bind
             ("v1",
              { Syntax.v = (Syntax.String "s");
                loc =
                { Syntax.fname = "test/cases/parse/object_for.in";
                  ran =
                  (Some { Syntax.startpos = { Syntax.line = 2; column = 13 };
                          endpos = { Syntax.line = 2; column = 16 } })
                  }
                }))
           ],
         { Syntax.v = (Syntax.Var "x");
           loc =
           { Syntax.fname = "test/cases/parse/object_for.in";
             ran =
             (Some { Syntax.startpos = { Syntax.line = 3; column = 3 };
                     endpos = { Syntax.line = 3; column = 4 } })
             }
           },
         { Syntax.v = (Syntax.String "y");
           loc =
           { Syntax.fname = "test/cases/parse/object_for.in";
             ran =
             (Some { Syntax.startpos = { Syntax.line = 3; column = 7 };
                     endpos = { Syntax.line = 3; column = 10 } })
             }
           },
         [(Syntax.Bind
             ("v2",
              { Syntax.v = (Syntax.String "s");
                loc =
                { Syntax.fname = "test/cases/parse/object_for.in";
                  ran =
                  (Some { Syntax.startpos = { Syntax.line = 4; column = 13 };
                          endpos = { Syntax.line = 4; column = 16 } })
                  }
                }))
           ],
         ("x",
          { Syntax.v = (Syntax.Var "a");
            loc =
            { Syntax.fname = "test/cases/parse/object_for.in";
              ran =
              (Some { Syntax.startpos = { Syntax.line = 5; column = 11 };
                      endpos = { Syntax.line = 5; column = 12 } })
              }
            }),
         [(Syntax.Forspec
             ("y",
              { Syntax.v = (Syntax.Var "a");
                loc =
                { Syntax.fname = "test/cases/parse/object_for.in";
                  ran =
                  (Some { Syntax.startpos = { Syntax.line = 6; column = 11 };
                          endpos = { Syntax.line = 6; column = 12 } })
                  }
                }));
           (Syntax.Ifspec
              { Syntax.v = Syntax.True;
                loc =
                { Syntax.fname = "test/cases/parse/object_for.in";
                  ran =
                  (Some { Syntax.startpos = { Syntax.line = 7; column = 5 };
                          endpos = { Syntax.line = 7; column = 9 } })
                  }
                })
           ])));
  loc =
  { Syntax.fname = "test/cases/parse/object_for.in";
    ran =
    (Some { Syntax.startpos = { Syntax.line = 1; column = 0 };
            endpos = { Syntax.line = 8; column = 1 } })
    }
  }